File: week_5/2.c
#include <stdio.h>
int main() {
    FILE *f;
    f = fopen("/Users/shogun/Desktop/ttest.txt", "w");
    printf("Input data string:\n");
    while (1) {
        char c[512];
        scanf("%s", c);
        fprintf(f, "%s\n", c);
        if (c[0] == '.' && c[1] == '\0') break;
    }
    fclose(f);
}

File: DATA STRUCTURES/6/2.py
def length(txt):
    n = -1
    u = ''
    if txt:
        s, *t = txt
        n, j = length(t)
        u = s + '+-'[n%2] + j
        
    return n+1, u

k, l = length(input('Enter Input : '))
v = '~*' if k%2 else '*~'
print(l.replace('+', v[1]).replace('-', v[0]))
print(k)